### ISSUE TITLE
Eliot algorithm raises error

### ISSUE THREAD

**Issue by user202729** [metebalci/pdftitle#47]
Document: http://matwbn.icm.edu.pl/ksiazki/aa/aa77/aa7747.pdf

Error:

```
[tmp]$ pdftitle -a eliot -p aa7747.pdf 
Traceback (most recent call last):
  File ".local/bin/pdftitle", line 8, in <module>
    sys.exit(run())
             ^^^^^
  File ".local/lib/python3.12/site-packages/pdftitle/pdftitle.py", line 551, in run
    eliot_tfs = args.eliot_tfs.split(",")
                ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'split'
```

Looks like an easy fix.

------

Although even with explicit value specified it has the spaces missing, not sure if this is easily handleable

```
[tmp]$ pdftitle --eliot-tfs=0 -a eliot -p aa7747.pdf 
Explicit-descentsonanellipticcurve
```

**Comment by metebalci** [comment]
Thanks for the report. Yes, the first one is just a simple bug to fix.

For the second, I created #48

**Comment by metebalci** [comment]
fixed with 0.17
